src/compiler/sys/resolve/resolve-module-sync.ts

```diff
@@ -139,9 +139,12 @@ export const createCustomResolverSync = (
       const fsFilePath = normalizeFsPath(p);
       try {
         return sys.realpathSync(fsFilePath);
```

Comment:
Unrelated to your changes... does this even throw an exception? According to the interface of `sys.realpathSync()` it returns a `CompilerSystemRealpathResults` and doesn't throw an exception:
https://github.com/ionic-team/stencil/blob/b00585fb643332eb9b1c188e8e23a8e8a4ec742d/src/declarations/stencil-public-compiler.ts#L1032-L1035

Reply:
It shouldn't, although I think the problem here is that we can't guarantee implementations on that interface _won't_ throw. Perhaps this is just a bit of defensive programming?